dookug-api/dookug-api-jee8/dookug-api-rest-jee8/src/main/java/hu/icellmobilsoft/dookug/api/rest/IOpenapiConstants.java

```diff
@@ -0,0 +1,77 @@
+/*-
+ * #%L
+ * DookuG
+ * %%
+ * Copyright (C) 2023 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.dookug.api.rest;
+
+/**
+ * Openapi constants for tags and descriptions
+ *
+ * @author mate.biro
+ * @since 0.1.0
+ */
+public interface IOpenapiConstants {
```

Comment:
Kicsit fura, hogy itt vannak bizonyos dolgok az openapi dokumentációhoz, bizonyos helyeken meg a rest interfacebe van írva. Ez nagyon inkonzisztens, ezt egységesre kellene hozni. Én személy szerint ezt a megoldást preferálom mert a leírások 1 helyen vannak, könnyebb fordítani, újrafelhasználható.

Reply:
ez is rendbe lesz majd hozva, de ebben az MR-ben csak at akartam strukturalni a projektet az uj 'formatumra', aztan persze ezek is javitva lesznek a vegen. (de igy legalabb fel is lesz irva :))